docs/technical-doc/api.rst

```diff
@@ -58,8 +58,6 @@ slot, timestamp), clique count, connected nodes count.
         "clique_count": Number,
         "end_timespan": Number,// stats time interval, millis since 1970-01-01
         "final_block_count": Number,
-        "final_operation_count": Number,
```

Comment:
Modify the openrpc instead. This has been removed on main.

Reply:
reverted